src/Utils.js

```diff
@@ -165,14 +165,31 @@ export function splitSelector(selector) {
   return selector.split(/(?=\.|\[.*\])|(?=#|\[#.*\])/);
 }
 
-export function isSimpleSelector(selector) {
-  // any of these characters pretty much guarantee it's a complex selector
-  return !/[~\s:>]/.test(selector);
+
+const containsQuotes = /'|"/;
+const containsColon = /:/;
+
+
+export function isPsuedoClassSelector(selector) {
+  if (containsColon.test(selector)) {
+    if (!containsQuotes.test(selector)) {
+      return true;
+    }
+    const tokens = selector.split(containsQuotes);
+    for (let i = 0; i < tokens.length; i++) {
+      const token = tokens[i];
+      if (containsColon.test(token) && i % 2 === 0) {
```

Comment:
seems like we could use `some` to do this without a loop? `return tokens.some((token, i) => containsColon.test(token) && i % 2 === 0)`?